src/utils/storage.ts

```diff
@@ -0,0 +1,319 @@
+import { GetInfoOutput } from '@arcana/auth-core'
+import dayjs from 'dayjs'
+
+import { AssetContract } from '@/models/Asset'
+import { NFT } from '@/models/NFT'
+
+enum StorageKey {
+  UserInfo = 'userInfo',
+  IsLoggedIn = 'isLoggedIn',
+  Session = 'session',
+  HasMFA = 'has-mfa',
+  SkipMFAUntil = 'mfa-skip-until',
+  MFADND = 'mfa-dnd',
+  LoginSrc = 'loginSrc',
+  LoginCount = 'login-count',
+  State = 'state',
+  NFT = 'nft_list',
+  AssetContract = 'asset-contracts',
+  PK = 'pk',
+}
+
+type UserInfo = GetInfoOutput & {
+  hasMfa?: boolean | undefined
+  pk?: string | undefined
+}
+
+type NFTItem = NFT & {
+  autodetected: boolean
+  chainId: number
+}
+
+function is3PCEnabled() {
```

Comment:
`are3PCEnabled`? What would ‘is 3rd party cookie enabled’  mean?

Reply:
@mmjee updated